config/webpack/loaders/devSitePackage.js

```diff
@@ -7,12 +7,23 @@ const loader = async function loader(content) {
   const callback = this.async();
 
   const json = JSON.parse(content);
+  const repoUrl = (json.repository && json.repository.url) ? (json.repository.url).match(/https.*[^.git]/i) : '';
+  const repoDirectory = (json.repository && json.repository.directory) ? json.repository.directory : '';
+  let finalUrl = '';
+  if (repoUrl) {
+    if (repoDirectory) {
+      finalUrl = `${repoUrl}/tree/master/${repoDirectory}`;
```

Comment:
what happens if repoDirectory is `/packages/my-package`? Does this result in `repoUrl/tree/master//packages/my-package`;

Reply:
Yes thats is correct.
Should we handle this by removing the extra front slash?

Reply:
yes should script out the extra `/`.  we shouldn't add broken links in

Reply:
Filtered out the extra front slash here : https://github.com/cerner/terra-dev-site/pull/243/commits/3acf9f958a72e6ad031400f2c566813e15a3c78d